src/main/java/hellos/Hellos.java

```diff
@@ -19,6 +19,7 @@ public void run() {
   public String generateOutput() {
     StringBuilder builder = new StringBuilder();
 
+    builder.append(
```

Comment:
This isn't a complete statement/line. You probably wanted:

```suggestion
    builder.append(aliceSaysHello());
```